package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-source-control-extension-tools",
-  "version": "4.1.12",
+  "version": "4.2.0",
```

Comment:
Can we bump the major version to 5.0? Since we are planning to deprecate the deploy-extensions, we can branch off here and only support 5.x of this package with `deploy-cli` and 4.x for the deploy-extensions. This should allow us to patch and release the 5.x faster w/o the limitation to deploy to webtask (and dependencies).